source/meetings/2022/november/index.html.md

```diff
@@ -0,0 +1,76 @@
+---
+updated_by:
+  email: alessandro.proserpio@lrug.org
+  name: Alessandro Proserpio
+created_by:
+  email: alessandro.proserpio@lrug.org
+  name: Alessandro Proserpio
+category: meeting
+title: November 2022 Meeting
+published_at: 2022-10-20 18:00:00 +0000
+created_at: 2022-10-20 18:00:00 +0000
+status: Published
+hosted_by:
+  - :name: Zappi
+
+---
+
+The November 2022 meeting of LRUG will be on *Monday the 14th of
+November*, from _6:00pm_ to _8:00pm_ (meeting starts at _6:30pm_).
+
+**👪 in person meeting alert 👪**
+
+We are keeping up our streak of in person events. The lovely folks at are
+hosting us in their offices
+For the first time since March 2020 we're having an in person event. The
```

Comment:
Probably want to tidy this paragraph up:
* there's a incomplete sentence at the start
* it's no longer the "first time since March 2020"

Reply:
Tidied this up myself when adding the sponsor logo.